providers/axfrddns/axfrddnsProvider.go

```diff
@@ -133,14 +135,22 @@ func initAxfrDdns(config map[string]string, providermeta json.RawMessage) (provi
 	if err != nil {
 		return nil, err
 	}
+	switch strings.ToLower(strings.TrimSpace(config["buggy-cname"])) {
+	case "yes", "true":
+		api.serverHasBuggyCNAME = true
+	default:
+		api.serverHasBuggyCNAME = false
+	}
 	for key := range config {
 		switch key {
 		case "master",
 			"nameservers",
 			"update-key",
 			"transfer-key",
 			"update-mode",
-			"transfer-mode":
+			"transfer-mode",
+			"domain",
+			"TYPE":
```

Comment:
Isn't buggy-cname missing here?
I tried setting it and I always get this logged:

```
[Warning] AXFRDDNS: unknown key in `creds.json` (buggy-cname)
```